### DISCUSSION TITLE
Only superusers can perform this action.    403

### DISCUSSION THREAD

**Discussion by lishaozheng** [pocketbase/pocketbase#6084] [Q&A]
The account I used is 1363435208@qq.com. Is there any problem?
![image](https://github.com/user-attachments/assets/cb125a6b-42fe-4cd3-8d76-4f16baaa8c5b)

![image](https://github.com/user-attachments/assets/68a6c738-8933-4eaf-994b-07269688f098)

**Comment by ganigeorgiev** [comment]
Make sure that the token in your `Authorization` header is for a superuser account (_you can inspect the JWT payload with [jwt.io](https://jwt.io/)_).

Based on the screenshot that's not the case because `data.auth` table field would have been `_superusers` and in your case is `users`.

**Reply by lishaozheng** [reply]
Thank you very much for your reply.
Why is neither of these accounts a super account.  
How can I obtain a super account?
![image](https://github.com/user-attachments/assets/d1be388d-d209-4752-b17f-b8dc695403ae)

**Reply by ganigeorgiev** [reply]
I don't understand what do you mean. Again, make sure that you are authenticating with account from the `_superusers` collection and not `users`, aka.:

```js
await pb.collection("_superusers").authWithPassword("EMAIL", "PASS")
```

If you need further help, please try to elaborate a little more on your project, what is the desired behavior and how you are sending the request in the first place.
